.buildkite/pipeline.yml

```
@@ -104,13 +111,16 @@ steps:
         matrix:
           - x64
           - arm64
-        notify:
-          - github_commit_status:
-              context: All Mac Dev Builds
-    if: build.tag !~ /^v[0-9]+/
+
+  - block: "🚦 Build for Windows?"
```

```suggestion
  - input: "🚦 Build for Windows?"
```
`block` steps are same as `input` except that they also implicitly add dependencies to all steps declared before it, and make all steps declared after it be dependent on it too. Which is not what we want here.
Instead we want `input` and declare our dependencies explicitly with `key`+`depends_on` (which we already do)

---

Thanks, updated. It seems that the chart shows the chain of dependencies correctly now.